src/cascadia/TerminalApp/lib/TerminalAppLib.vcxproj

```diff
@@ -271,7 +271,7 @@
   <ItemDefinitionGroup>
     <ClCompile>
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
-      <AdditionalIncludeDirectories>..;$(OpenConsoleDir)\dep;$(OpenConsoleDir)\dep\jsoncpp\json;%(AdditionalIncludeDirectories);</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>..;$(OpenConsoleDir)\dep\jsoncpp\json;%(AdditionalIncludeDirectories);</AdditionalIncludeDirectories>

```

Comment:
CLI11.hpp won't be accessible to TerminalAppLib pch.h that this change added https://github.com/microsoft/terminal/commit/830c22b73e3299cde552208597ddebffbeb0a898

Reply:
`dep` was always on the include line, so adding it _earlier_ than `AdditionalIncludeDirectories` was simply done in error 😄

Reply:
Ah 🙂